_includes/parse-server/database.md

```diff
@@ -44,8 +40,8 @@ Details about the configuration options can be found on [pg-promise](https://git
 * You will need to configure a [file adapter](#configuring-file-adapters) in order to store files.
 * Join tables are resolved in memory, there is no performance improvements using Postgres over MongoDB for relations or pointers.
 * Mutating the schema implies running ALTER TABLE, therefore we recommend you setup your schema when your tables are not full.
-* The postgres URL for 4.2.0 and below only supports the following configuration options:
+* The Postgres URL for Parse 4.2.0 and below only supports the following configuration options:
```

Comment:
More specific:

> The Postgres URL for Parse Server 4.2.0

because "Parse 4.2.0" could be understood as the Parse JS SDK.